includes/class-wsuwp-shortcode-thankyoupage.php

```diff
@@ -85,6 +85,19 @@ public function thankyoupage_create_shortcode( $atts, $content = null ) {
 		$content = $this->replace_conditionals( $content, $query_params );
 
 		foreach ( $query_params as $key => $value ) {
+			if ( ( 'donor.id' === $key || 'transaction_id' === $key ) && $value ) {
+				$value = strtoupper( substr( $value, 0, 8 ) );
+			} elseif ( 'created' === $key && $value ) {
+				date_default_timezone_set( 'America/Los_Angeles' );
```

Comment:
Don't set a default timezone. We either need to show in the local timezone or hide the timezone field altogether.